source/s3_meta_request.c

```diff
@@ -1781,7 +1843,21 @@ static void s_s3_meta_request_event_delivery_task(struct aws_task *task, void *a
                     if (meta_request->meta_request_level_running_response_sum) {
                         aws_checksum_update(meta_request->meta_request_level_running_response_sum, &response_body);
```

Comment:
I know this isn't part of your PR. But I see 😬 we're not checking aws_checksum_update() for error